shacl/index.html

```diff
@@ -609,17 +609,17 @@ <h1 id="part1" style="font-size: 160%; font-weight: bold">Part 1: Core Features<
 				<ul>
 					<li><a href="#scopeNode"><b>Node scopes</b></a> define a specific RDF node as scope.</li>
 					<li><a href="#scopeClass"><b>Class-based scopes</b></a> define the scope as the set of all <span class="term"><a href="#shacl-rdfs">instances</a></span> of a class.</li><!--TODO: check instance -->
-					<li><a href="#scope"><b>General scopes</b></a> define a flexible mechanism to produce arbitrary focus nodes.</li>
+					<li><a href="#scope"><b>General scopes</b></a> are a flexible mechanism to define arbitrary focus nodes.</li>
 				</ul>
 				<p>
 					When multiple scopes are provided in a shape, the scope of a shape is the union of all focus nodes produced by these scopes.
-					Focus nodes produced by scopes may not exist in the data graph.
+					Focus nodes produced by scopes might not exist in the data graph.
 				</p>
 				
 				<section id="scopeNode">
 					<h4>Node scopes (sh:scopeNode)</h4>
 					<p>
-						Shapes define node scopes with the <code>sh:scopeNode</code> predicate.
+						Shape properties define node scopes with the <code>sh:scopeNode</code> predicate.
```

Comment:
This is back to the "shapes do not act" problem. A shape is a thing (an 
IRI) that has a scope. What gives it that scope? Perhaps we can turn 
this around and say "node scopes are defined with the 
<code>sh:scopeNode</code> predicate". Or "the scope of a node is defined 
with ..." ("node scopes" is pretty awkward). Basically, predicates are 
verbs and can "do" something; subjects are nouns and do not act.

kc

On 4/24/16 6:49 AM, Dimitris Kontokostas wrote:
> In shacl/index.html
> <https://github.com/w3c/data-shapes/pull/18#discussion_r60844890>:
>
>>  				</p>
>>  				
>>  				<section id="scopeNode">
>>  					<h4>Node scopes (sh:scopeNode)</h4>
>>  					<p>
>> -						Shapes define node scopes with the <code>sh:scopeNode</code> predicate.
>> +						Shape properties define node scopes with the <code>sh:scopeNode</code> predicate.
>
> I find the term "properties" confusing in this sentence.
> it is the shape that has a scope (not the shape properties) and
> depending on the scope we use a different predicate (property) to define it.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/w3c/data-shapes/pull/18/files/52fded8308a039314d783a385110ed702c951dcb#r60844890>
>

-- 
Karen Coyle
kcoyle@kcoyle.net http://kcoyle.net
m: 1-510-435-8234
skype: kcoylenet/+1-510-984-3600